workflow-runtime/src/iosMain/kotlin/com.squareup.workflow1.internal/SystemUtils.kt

```diff
@@ -0,0 +1,6 @@
+package com.squareup.workflow1.internal
+
+import platform.Foundation.NSDate
+import platform.Foundation.timeIntervalSince1970
+
+actual fun currentTimeMillis(): Long = (NSDate().timeIntervalSince1970 * 1000).toLong()
```

Comment:
Should this be `CACurrentMediaTime()` instead? https://stackoverflow.com/questions/358207/iphone-how-to-get-current-milliseconds

> [timeIntervalSince1970] can return an earlier value when you call it after couple of miliseconds (e.g. call it consecutively and you might not end up with an increasing sequence)

Would require a dependency on `QuartzCore`, don't know if that's okay.

Reply:
I'm not sure. We would need specific iOS focused validation of this runtime anyway.